pkg/infra/log/requestTiming.go

```diff
@@ -0,0 +1,28 @@
+package log
+
+import (
+	"context"
+	"time"
+)
+
+type requestStartTimeContextKey struct{}
+
+var requestStartTime = requestStartTimeContextKey{}
+
+// InitCounter creates a pointer on the context that can be incremented later
+func InitstartTime(ctx context.Context, now time.Time) context.Context {
```

Comment:
Nit: Do we need to pass time here?

Reply:
I figured it would be easier to test.. all thou I didn't write any tests :angel: